### PR TITLE
Adding a achievement notification instead of the old thing

### PR DESCRIPTION
**Pull request by MAJigsaw77**
is tested don't think about i didn't did that, just take a look yourself @ShadowMario
Screenshot (153)
it looks similar to indie cross achievement notification system

### CODE REVIEW

**Comment by An-undertale-fan** [comment]
So this is the indie cross achievement thingie
Look great the normal psych achievement thing is boring 😒

**Comment by Swapper1** [comment]
This look better than the old one that can wait for 2 second until the next song or send back to Story/Freeplay menu. Funkin' Approved.

**Review by ghost — APPROVED** [review]

File: source/MainMenuState.hx [hunk 1]
```diff
@@ -251,7 +230,7 @@ class MainMenuState extends MusicBeatState
 					});
 				}
 			}
-			#if desktop
+			#if MODS_ALLOWED
```
Comment (RafPlayz69YT) [inline source/MainMenuState.hx:233]: Uh, it seems kinda dumb to do this, what if you want to edit characters quickly?????

Comment (jigsaw-4277821) [reply source/MainMenuState.hx:233]: You are right

**Pull request closed without merging**